Index: lib/screens/expenses_list_screen.dart
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- lib/screens/expenses_list_screen.dart	(date 1613150319000)
+++ lib/screens/expenses_list_screen.dart	(date 1613150678000)
@@ -1,5 +1,3 @@
-import 'dart:io';
-
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:gelir_gider/providers/expense_provider.dart';
@@ -7,9 +5,7 @@
 import 'package:gelir_gider/widgets/expense_item.dart';
 import 'package:gelir_gider/widgets/money_widget.dart';
 import 'package:provider/provider.dart';
-import 'package:theme_manager/theme_manager.dart';
-import 'package:theme_manager/change_theme_widget.dart';
-
+import 'package:gelir_gider/widgets/theme_dialog_widget.dart';
 import 'adding_expense_screen.dart';
 
 class ExpensesListScreen extends StatefulWidget {
@@ -18,21 +14,12 @@
 }
 
 class _ExpensesListScreenState extends State<ExpensesListScreen> {
-  void setAsSystemDefault() => ThemeManager.of(context)
-      .setBrightnessPreference(BrightnessPreference.system);
-  void setAsLight() => ThemeManager.of(context)
-      .setBrightnessPreference(BrightnessPreference.light);
-  void setAsDark() => ThemeManager.of(context)
-      .setBrightnessPreference(BrightnessPreference.dark);
-  void showThemePicker() {
+
+  void showThemePicker(bool isEnglish) {
     showDialog<void>(
       context: context,
       builder: (BuildContext context) {
-        return ThemePickerDialog(
-          onSelectedTheme: (BrightnessPreference preference) {
-            ThemeManager.of(context).setBrightnessPreference(preference);
-          },
-        );
+        return ThemeDialogWidget(isEnglish);
       },
     );
   }
@@ -40,36 +27,37 @@
   @override
   Widget build(BuildContext context) {
     final langState = Provider.of<LanguageHandler>(context, listen: false);
-
     return SafeArea(
       child: Scaffold(
         appBar: AppBar(
           title: Text(langState.isEnglish ? 'Add/Remove' : 'Ekle/Çıkar'),
           actions: [
             FlatButton(
-              onPressed: showThemePicker,
-              child: Icon(
-                Icons.color_lens,
-                color: Colors.white,
-              ),
-            ),
-          ],
+              onPressed: (){
+                showThemePicker(langState.isEnglish);
+              },
+          child: Icon(
+        Icons.color_lens,
+        color: Theme.of(context).buttonColor,
+        ),
+        ),
+        ],
         ),
         body: FutureBuilder(
-          future: Provider.of<Expenses>(context, listen: false)
-              .fetchAndSetExpenses(),
-          builder: (ctx, snapshot) => snapshot.connectionState ==
-                  ConnectionState.waiting
-              ? Center(
-                  child: CircularProgressIndicator(),
-                )
-              : Consumer<Expenses>(
-                  child: Center(
-                    child: langState.isEnglish
-                        ? const Text(
-                            'Got no Expenses  yet \n Start adding some !')
+        future: Provider.of<Expenses>(context, listen: false)
+            .fetchAndSetExpenses(),
+        builder: (ctx, snapshot) => snapshot.connectionState ==
+        ConnectionState.waiting
+        ? Center(
+        child: CircularProgressIndicator(),
+        )
+            : Consumer<Expenses>(
+        child: Center(
+        child: langState.isEnglish
+        ? const Text(
+                            'Got no Expenses  yet \nStart adding some !')
                         : const Text(
-                            'Henüz ekleme yapılmadı \n Eklemeye başlayın !'),
+                            'Henüz ekleme yapılmadı \nEklemeye başlayın !'),
                   ),
                   builder: (ctx, expenseProvider, ch) => expenseProvider
                           .expense.isEmpty
@@ -81,6 +69,7 @@
                               children: [
                                 Divider(
                                   height: 25,
+                                  color:Theme.of(context).accentColor,
                                 ),
                                 MoneyWidget(
                                   expense: expenseProvider
@@ -95,6 +84,7 @@
                                 ),
                                 Divider(
                                   height: 25,
+                                  color:Theme.of(context).accentColor,
                                 ),
                                 Flexible(
                                   child: ListView.builder(
